framework/test/mocha/unit/components/system/system.js

```diff
@@ -160,37 +160,83 @@ describe('components: system', () => {
 	});
 
 	describe('protocolVersionCompatible', () => {
-		describe('when protocol version is exactly equal to system protocol version', () => {
-			it('should return true', async () =>
-				expect(systemComponent.protocolVersionCompatible('1.0')).to.be.true);
-		});
-		describe('when the hard part of protocol is not exactly equal than the one of the system protocol version', () => {
-			it("should return false if it's greater or lesser", async () =>
-				expect(systemComponent.protocolVersionCompatible('2.0')).to.be.false);
-			it("should return false if it's lesser", async () =>
-				expect(systemComponent.protocolVersionCompatible('0.0')).to.be.false);
-		});
-		describe('when the hard part of protocol is equal to  the one of the system protocol version', () => {
-			it('should return true', async () =>
-				expect(systemComponent.protocolVersionCompatible('1.5')).to.be.true);
+		describe('when the hard for part has one digit', () => {
+			beforeEach(async () => {
+				systemComponent.headers.protocolVersion = '1.0';
+			});
+
+			describe('when protocol version is exactly equal to system protocol version', () => {
+				it('should return true', async () =>
+					expect(systemComponent.protocolVersionCompatible('1.0')).to.be.true);
+			});
+			describe('when the hard part of protocol is not exactly equal than the one of the system protocol version', () => {
```

Comment:
Could be simplified to => `when the hard part of the peer protocol is not exactly equal to the system protocol version`